DMTN-266.tex

```diff
@@ -48,6 +49,178 @@
+\begin{equation}
+\Delta \symbf{x_i} = \symbf{D}_{visit} \; c_m
+\end{equation}
+where $\symbf{D}_{visit}$ is the DCR coefficient determined by the pointing and atmospheric conditions of a given exposure and $c$ is the color parameter for object $m$. 
+
+The main driver in variability of the DCR coefficient is the pointing, so $\symbf{D}_{visit}$ can be modeled generally by the equation
+\begin{equation}
+\symbf{D}_{visit} = D_{band} \; \mathrm{tan}(z) \symbf{\hat{p}}
+\end{equation}
+where $z$ is the zenith distance, $\symbf{\hat{p}}$ is the unit sky vector towards zenith, and $D_{band}$ is a constant per-band that is fit to the ensemble of observations. Figure~\ref{fig:dcr} shows the linear relationship between the $D_visit$ fit per visit and the pointing, using simulated ComCam data.
+\begin{figure}
+\includegraphics[width=\columnwidth]{figures/dcr_demo.png}
+\caption{Fit values for $D_{band}$ as a function of the telescope pointing. }
+\label{fig:dcr}
+\end{figure}
+However, given the large number of sources per visit that are expected from LSST, we can likely get a better fit of the effects of DCR by fitting $D_{visit}$ per exposure in order to allow for time-dependent atmospheric effects.
+
+The default color used is $g - i$, but this is configurable in the task. The magnitudes used to calculate the color come from the \texttt{fgcmCal} calibration of isolated stars, meaning that colors are not available for all objects due to selection differences. %Figure <TODO: figure here> shows the fit residuals as a function of color at a range of airmasses, using simulated ComCam data.
+
+
+
+\section{Fitting Modes}\label{sec:modes}
+The baseline mode for fitting the astrometric model, as described in Section~\ref{sec:final}, is to fit all of the visits in a given band that are overlapping with a given tract of the sky. However, the model fitting can objectively be improved by using larger amounts of data, and there are scenarios, which will be common for ComCam and in the early days of LSST, where there will not be enough data per tract in a given band. Because of this, a number of different running modes have been developed with an eye towards what will be needed during commissioning.
+
+\subsection{Multiband fitting}
+The astrometric model is expected to be different from band to band due to differences in transmission through both the atmosphere and the telescope optics and camera. However, the true positions of the sources are obviously the same regardless of the filter through which they are observed. Fitting multiple band simultaneously thus provides a stronger constraint on this aspect of the model and leads to a fit with greater precision. Running with multiple bands is done by the task \texttt{GbdesAstrometricMultibandFitTask}, which inherits from \texttt{GbdesAstrometricMultibandFitTask} and is algorithmically identical. Note that this is independent from the DCR fitting described in Section~\ref{sec:dcr}.
+
```

Comment:
You have the task inheriting from itself here, so one of these is probably a typo.